cardano-sync/src/Cardano/Sync.hs

```diff
@@ -276,9 +268,9 @@ dbSyncProtocols trce env metricsSetters plugin queryVar runDBThreadFunction vers
     localStateQuery :: RunMiniProtocol 'InitiatorMode BSL.ByteString IO () Void
     localStateQuery =
       InitiatorProtocolOnly $ MuxPeer
-        (contramap (Text.pack . show) . toLogObject $ appendName "local-state-query" trce)
+        Logging.nullTracer
         (cStateQueryCodec codecs)
-        (localStateQueryClientPeer (localStateQueryHandler queryVar))
+        localStateQueryPeerNull
```

Comment:
Why? I suspect there is useful logging coming out of those two loggers.

Reply:
we no longer use the local state query